tests/components/alarm_control_panel/test_manual.py

```diff
@@ -1319,3 +1319,51 @@ def test_arm_away_after_disabled_disarmed(self):
 
         state = self.hass.states.get(entity_id)
         self.assertEqual(STATE_ALARM_TRIGGERED, state.state)
+
+
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax. Please drop this decorator and prefix async methods with `async` like `async def test_restore_armed_state `. Replace any `yield from` with `await`.

Reply:
I've change the syntax as requested. I based my tests on the tests i found in the input variable components, those are still using the asyncio decorator.

Reply:
I know. Those need to be updated too. For now we just make sure we don't add new ones.